package.json

```diff
@@ -25,13 +25,16 @@
     "chromatic": "npm --prefix examples/official-storybook run chromatic",
     "coverage": "codecov",
     "danger": "danger",
-    "dev": "concurrently --kill-others \"yarn dev:ts\" \"yarn dev:js\"",
-    "dev:js": "lerna exec --parallel -- cross-env-shell node \\$LERNA_ROOT_PATH/scripts/watch-js.js",
-    "dev:ts": "lerna exec --parallel -- cross-env-shell node \\$LERNA_ROOT_PATH/scripts/watch-ts.js",
+    "dev": "concurrently --kill-others \"yarn dev:tsc\" \"yarn dev:babel\"",
```

Comment:
@kroeder Why does `yarn dev` still spawn `yarn dev:tsc`? I thought the major selling point of this PR was that we wouldn't have to run both?

Reply:
shouldn't this be `dev:babel` + `dev:check=types` and only `dev:tsc` for angular?

Reply:
Dev:tsc creates definition files and compiles angular, check types just checks types with --noEmit

Reply:
@shilman  In a green dream world I tried to just run `tsc` after each babel-compile.js run which means there are only 50 instead of 100 watch-processes (which is insane 💥 ) but I noticed that I cannot hook on `babel --watch` so `tsc --watch` needs to be started as well for generating `d.ts` files

I thought about to use a custom file watcher that just starts `babel` and `tsc` without `--watch` but I was worried about the performance because babel + tsc would need a cold-start without `--watch`. `tsc` has a cache since 3.4 but I'm not sure how babel handles cold starts

Ultimately, I thought this can be step 2 in improving the performance and I wanted to finish this first step since it already took me a lot of time to do this change

Reply:
Implementing a watcher ourselves would be a bad idea.

We should make sure that both tsc & babel watch as few files as possible

Reply:
Fun fact: babel uses `gulp watch`
What transpiles babel? 😮 🙂